lib/tty.js

```diff
@@ -53,7 +53,7 @@ function WriteStream(fd) {
   // this behaviour has become expected due historical functionality on OS X,
   // even though it was originally intended to change in v1.0.2 (Libuv 1.2.1).
   // Ref: https://github.com/nodejs/node/pull/1771#issuecomment-119351671
-  this._handle.setBlocking(process.env.NODE_TTY_UNSAFE_ASYNC !== '1');
+  this._handle.setBlocking(true);
```

Comment:
@Fishrock123 Why do we do this here, for all TTY streams? Shouldn’t we just do it for `process.stderr`/`process.stdin`?